drivers/adc/ad7768-1/ad77681.c

```diff
@@ -324,6 +324,57 @@ int32_t ad77681_spi_read_adc_data(struct ad77681_dev *dev,
 	return ret;
 }
 
+/**
+ * Read continuous conversion result from device.
+ * @param dev - The device structure.
+ * @param adc_data - The conversion result data
+ * @return 0 in case of success, negative error code otherwise.
+ */
+int32_t ad77681_spi_read_adc_cont_data(struct ad77681_dev *dev,
```

Comment:
This functions is very similar to ad77681_spi_read_adc_data.
Can you create just one function and control with one parameter if it is continuous read?
This function can be internal to the driver, an both ad77681_spi_read_adc_data and ad77681_spi_read_adc_cont_data call it

Reply:
I think it would be better to have a separate "ad77681_spi_read_adc_cont_data " function. This function will be called thousand times in a seconds when reading the continuously streamed data from the ADC. Merging the functions, the controller would also check thousand of times in a sec the type of read (register or continuous read) will be performed if operating on a continuous read mode.

Reply:
I see, I think from the maintenance point of view will be good to don't have duplicated code. A good example is https://github.com/analogdevicesinc/no-OS/pull/957/commits/7bd24bfc17d3a92128f3073fb29830e46c7e2f43#r589349134
.
Anyway maybe someone else of @analogdevicesinc/noos team have other opinion.

Reply:
I will agree with Mihail here. The function could be `inline` if call times are an issue.

Reply:
Done merging two functions. Please check if the implementation is okay.